src/components/tabs/components/tab-button-controls/tab-button-controls.module.css

```diff
@@ -4,9 +4,22 @@ the correct elements, roles, states, and properties are being set in the markup.
 */
 
 .tabList[role='tablist'] {
+  position: relative;
   align-items: center;
   display: flex;
   gap: 2px;
+
+  /* In nested tabs, include an underline-ish style under tab buttons */
+  &.isNested::after {
+    background: var(--token-color-border-primary);
+    bottom: 0;
+    content: '';
+    height: 1px;
+    left: 0;
+    position: absolute;
+    right: 0;
+    z-index: -1;
+  }
```

Comment:
❓ Is it possible to use a normal CSS border here instead of a pseudo element?

Reply:
I don't think so, at least without changing a bunch of other stuff - added a comment to clarify why. Happy to dive deeper here if we want to strictly avoid this 🙇